app/src/androidTest/java/com/hedvig/app/feature/offer/StartDateFromPreviousInsurerTest.kt

```diff
@@ -60,4 +60,25 @@ class StartDateFromPreviousInsurerTest : TestCase() {
             }
         }
     }
+
+    @Test
+    fun previousInsurerNameShouldShowInSwitch() = run {
+        activityRule.launch(OfferActivity.newInstance(context(), listOf("123")))
+        OfferScreen {
+            scroll {
+                childAt<OfferScreen.HeaderItem>(0) {
+                    startDate {
+                        click()
+                    }
+                }
+            }
+        }
+        val text = context().getString(R.string.OFFER_PLAN_EXIRES_TEXT, "Test current insurer")
+
+        ChangeDateView {
+            switches {
+                hasText(text)
+            }
+        }
```

Comment:
We have an extension for this in `KakaoExtensions` 🙂 also, should we refer to the test data instead of hard-coding the `Test current insurer`?

```suggestion
        ChangeDateView {
            switches {
                hasText(R.string.OFFER_PLAN_EXIRES_TEXT, "Test current insurer")
            }
        }
```